ce/api/percentileanomaly.py

```diff
@@ -240,6 +240,34 @@ def canonical_timestamp(timestamp, timescale, timeidx):
                 projected_data[timescale][timestamp] = list(
                     np.percentile(anomalies, percentiles))
 
+        # converts a nested dictionary to a list storing data objects
```

Comment:
Good comment to describe what the function does. It would be even better if you used this comment as [the function's docstring](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings).